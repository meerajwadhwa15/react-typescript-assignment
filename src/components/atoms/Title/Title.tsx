import React, { FunctionComponent, ReactNode } from 'react'

export type Props = {
  children: ReactNode
}

const Img: FunctionComponent<Props> = ({ children }) => <h1>{children}</h1>

export default Img
