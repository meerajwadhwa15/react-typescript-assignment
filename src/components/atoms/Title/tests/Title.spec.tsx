import React from 'react'
import { shallow } from 'enzyme'
import { create } from 'react-test-renderer'
import Title, { Props } from '../Title'

const props: Props = {
  children: 'test',
}

describe('Title Enzyme testing', () => {
  test('renders without crashing', () => {
    shallow(<Title {...props} />)
  })
})

describe('Title Snapshot testing', () => {
  test('Matches the snapshot', () => {
    const component = create(<Title {...props} />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
