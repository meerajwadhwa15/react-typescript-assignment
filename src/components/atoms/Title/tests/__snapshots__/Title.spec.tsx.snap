// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Snapshot testing Matches the snapshot 1`] = `
<h1>
  test
</h1>
`;
